### ISSUE TITLE
ceremony progress web view

### ISSUE THREAD

**Issue by redshiftzero** [penumbra-zone/penumbra#3220]
Now that we have the pieces for the ceremony, it would be nice to have a web view that displays the progress of the ceremony, including the:
* Number of contributions thus far
* Current length of contribution queue (i.e. the number of people currently connected waiting to contribution)
* Last winning bid

See example code here: https://github.com/tokio-rs/axum/blob/main/examples/templates/src/main.rs
And relevant discussion in Discord starting [here](https://discord.com/channels/824484045370818580/948673930909077524/1163620197689720893)

**Comment by cronokirby** [comment]
c.f. https://github.com/penumbra-zone/penumbra/issues/3221 for the refactoring mentioned here.

**Comment by hdevalence** [comment]
#3225 unblocks this.

**Comment by cronokirby** [comment]
https://github.com/penumbra-zone/penumbra/issues/3230 for tracking the schema change specifically
